src/fu-security-attr.c

```diff
@@ -179,6 +179,10 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
 		 * debugging of Intel processors using the USB3 port */
 		return g_strdup(_("Intel DCI debugger"));
 	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUPPORTED_SOC) == 0) {
+		/* TRANSLATORS: Title: if fwupd supports HSI on this chip */
+		return g_strdup(_("Supported SoC"));
```

Comment:
I think most people know what a CPU is, and soc is probably much less understood.